src/SIM/SIMbase.h

```diff
@@ -730,6 +734,7 @@ class SIMbase : public SIMinput, public SIMdependency
   IntegrandBase* myProblem; //!< The main integrand of this simulator
   IntegrandMap   myInts;    //!< Set of all integrands involved
   AnaSol*        mySol;     //!< Analytical/Exact solution
+  ModelGenerator* myGen;    //!< Model generator to use
```

Comment:
Mulig jeg ikke ser det heller, men trenger ModelGenerator-objektet egentlig å være en member? Jeg trodde det bare var en helper som trenger å eksistere kun ved parsing av input-filen, for så å generere den faktiske modellen etter oppskriften, og deretter var den ferdig med sitt. Dvs. den lever kun inne i "parse geometry" blokka.